spec/lita/handler/state_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "spec_helper"
+
+describe Lita::State do
+  describe "#initialize" do
+    it "instatiates" do
+      expect(described_class.new).to be_a Lita::State
+    end
+  end
+
+  let!(:storage) { described_class.new }
+  let!(:data) { OpenStruct.new(best_bot: "Litabot!") }
+
+  before :each do
+    dat = data
+    storage.set(:expensive_key, dat)
+  end
```

Comment:
All this setup is not necessary—let's just have one example for `#get` and one example for `#set`. (Remove the let blocks and the before block.)